public/components/security/roles-mapping/roles-mapping-table.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Changes the indentation of the document to two spaces

And remove the `useState` and the `useEffect`, you never use them.